tests/functional/test_dashboard.py

```diff
@@ -63,14 +73,19 @@ def test_dashboard_overflow(base_url, selenium):
 
     bug 1405690 - some content causes overflows
     """
+    admin = AdminLogin(selenium, base_url).open()
+    admin.login_new_user()
     page = DashboardPage(selenium, base_url).open()
     page.open_first_details()
     assert page.scroll_width <= page.client_width
 
 
 @pytest.mark.nondestructive
+@pytest.mark.login
 @skip_if_not_maintenance_mode
 def test_dashboard_in_mm(base_url, selenium):
```

Comment:
This test (`test_dashboard_in_mm`) you can delete now, since in maintenance mode all logins (even admin logins) are disabled and the test would fail.

Reply:
Thanks for noticing that. Done.